virttest/libvirt_xml/devices/interface.py

```diff
@@ -87,12 +87,11 @@ def __init__(self, type_name, virsh_instance=base.base.virsh):
                                parent_xpath='/',
                                tag_name='model',
                                attribute='type')
-        accessors.XMLAttribute(property_name="coalesce",
```

Comment:
do we have interface type as coalesce, I see it is part of network interface type from,
https://libvirt.org/formatdomain.html#coalesce

Reply:
Yes, libvirt has support coalesce. 
This commit is merely to extend coalesce/rx/frame to support more attributes not only 'max'. InterfaceObj.coalesce = {'max':coalesce_value}

Until now, only max attribute is defined in coalesce/rx/frame. Further future, if new attribute is defined, coalesce in avocado-vt doesn't need to change, only use dict directly in tp-libvirt, like
InterfaceObj.coalesce = {'max':coalesce_value, 'newattribute': value}